tst/EnergyPlus/unit/SurfaceGeometry.unit.cc

```diff
@@ -14591,3 +14591,1524 @@ TEST_F(EnergyPlusFixture, ExtSolarForShadingTest)
     EXPECT_FALSE(thisSG->SurfaceTmp(8).HeatTransSurf);
     EXPECT_TRUE(thisSG->SurfaceTmp(8).ExtSolar);
 }
+TEST_F(EnergyPlusFixture, SurfaceGeometry_ZoneOutsideBC_SpacesNotInput)
```

Comment:
3 new unit test to exercise various combinations of zone and space BC.

Reply:
Excellence.